### DISCUSSION TITLE
marianas: how take over bases?

### DISCUSSION THREAD

**Discussion by HILOK** [Khopa/dcs_liberation#1791] [Q&A] [answered]
this refers to V5.x / PACIFIC REPARTEE campaign; as title says, what are the game mechanics to take over a FOB or the red carrier?

more specific,
* is it required to destruct every single red unit attached to a node or just within a certain radius of the node? 
* must blue ground units or troops be deployed on the red islands to capture a base?
* if required, any recommendation how to kill the red carrier?

thanks

**Comment by Starfire13** [comment, marked as answer]
You can't capture carriers. You need ground units to capture FOBs. If the campaign doesn't provide a way to obtain ground units, then it is simply an A2G campaign and you are not expected to capture anything. You can kill carriers with Harpoons.

**Comment by HILOK** [comment]
alright, i'll try this, thanks
